src/main/java/com/agilethought/internship/sso/controller/ControllerApplication.java

```diff
@@ -0,0 +1,52 @@
+package com.agilethought.internship.sso.controller;
+
+import com.agilethought.internship.sso.domain.NewUserRequest;
+import com.agilethought.internship.sso.domain.NewUserResponse;
+import com.agilethought.internship.sso.domain.UpdateUserRequest;
+import com.agilethought.internship.sso.domain.UpdateUserResponse;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import com.agilethought.internship.sso.domain.UserDTO;
+import com.agilethought.internship.sso.services.ServiceApplication;
+import java.util.List;
+
+@RestController
+@Slf4j
+public class ControllerApplication {
+
+	@Autowired
+	ServiceApplication serviceApplication;
+
+	@PostMapping(value = "/api/v1/user", produces = "application/json")
```

Comment:
As far I remember we include @RequestMapping tag to define base path, is that rigth?